x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/AggregateMapper.java

```diff
@@ -278,6 +291,9 @@ private static String dataTypeToString(DataType type, Class<?> aggClass) {
         if (aggClass == ToPartial.class || aggClass == FromPartial.class) {
             return "";
         }
+        if ((aggClass == Max.class || aggClass == Min.class) && type.equals(DataType.IP)) {
```

Comment:
I would like to remove this class, and move the logic to the agg itself (Max.java). But not in this PR, I'll think about it and do it in another one